src/main/resources/application.properties

```diff
@@ -3,4 +3,9 @@ spring.datasource.driverClassName=org.h2.Driver
 spring.datasource.username=sa
 spring.datasource.password=password
 spring.jpa.database-platform=org.hibernate.dialect.H2Dialect
-spring.jpa.hibernate.ddl-auto=update
\ No newline at end of file
+spring.jpa.defer-datasource-initialization=true
+spring.sql.init.mode=always
+spring.jpa.hibernate.ddl-auto=update
+spring.mvc.pathmatch.matching-strategy = ANT_PATH_MATCHER
```

Comment:
What is the purpose of this line? Just curious

Reply:
For swagger to work.